types/components/FormCheck.d.ts

```diff
@@ -16,6 +16,7 @@ export interface FormCheckProps {
   isValid?: boolean;
   isInvalid?: boolean;
   feedback?: React.ReactNode;
+  inputAs?: React.ElementType;
```

Comment:
ah, crud, looking at the typing below, this actually _should_ be `as`, not `inputAs`

my bad